src/modules/components/proposal/proposalActions/actions/proposalActionChangeMembers/proposalActionChangeMembers.tsx

```diff
@@ -0,0 +1,64 @@
+import { DataList, Heading } from '../../../../../../core';
+import { MemberDataListItem } from '../../../../member';
+import { useOdsModulesContext } from '../../../../odsModulesProvider';
+import { ProposalActionType } from '../../proposalActionsTypes';
+import type { IProposalActionChangeMembers } from '../../proposalActionsTypes/proposalActionChangeMembers';
+
+export interface IProposalActionChangeMembersProps {
+    /**
+     * The action to render for Member count adjustment
+     */
+    action: IProposalActionChangeMembers;
+}
+
+export const ProposalActionChangeMembers: React.FC<IProposalActionChangeMembersProps> = (props) => {
+    const { action } = props;
+    const { copy } = useOdsModulesContext();
+    return (
+        <div className="flex flex-col gap-y-6">
+            <DataList.Root entityLabel="Members">
+                <DataList.Container SkeletonElement={MemberDataListItem.Skeleton}>
```

Comment:
We won't display a loading state here, we don't need to specify a skeleton element
```suggestion
                <DataList.Container>
```